src/components/Avatar/Avatar.tsx

```diff
@@ -125,34 +114,35 @@ export function Avatar({
   // Use `dominant-baseline: central` instead of `dy` when Edge supports it.
   const verticalOffset = '0.35em';
 
-  const initialsMarkup =
-    initials && !hasImage ? (
-      <span className={styles.Initials}>
-        <svg className={styles.Svg} viewBox="0 0 48 48">
-          <text
-            x="50%"
-            y="50%"
-            dy={verticalOffset}
-            fill="currentColor"
-            fontSize="26"
-            textAnchor="middle"
-          >
-            {initials}
-          </text>
-        </svg>
-      </span>
-    ) : null;
+  const avatarBody = customer ? (
+    <g fill="currentColor" fillRule="nonzero">
```

Comment:
Hey @kyledurand – as I was looking at the component, I noticed this minor refactor opportunity:

```diff
- <g fill="currentColor" fillRule="nonzero">
-   <path d="M8.28 27.5A14.95 14.95 0 0120 21.8c4.76 0 8.97 2.24 11.72 5.7a14.02 14.02 0 01-8.25 5.91 14.82 14.82 0 01-6.94 0 14.02 14.02 0 01-8.25-5.9zM13.99 12.78a6.02 6.02 0 1112.03 0 6.02 6.02 0 01-12.03 0z" />
- </g>
+ <path fill="currentColor" d="M8.28 27.5A14.95 14.95 0 0120 21.8c4.76 0 8.97 2.24 11.72 5.7a14.02 14.02 0 01-8.25 5.91 14.82 14.82 0 01-6.94 0 14.02 14.02 0 01-8.25-5.9zM13.99 12.78a6.02 6.02 0 1112.03 0 6.02 6.02 0 01-12.03 0z" />
```

Not a huge win, but I thought I'd bring it up in case you go back to this at some point!

Reply:
Nice find! I have another Avatar PR up so I just made the change there: https://github.com/Shopify/polaris-react/pull/2693/commits/72f899765a4e188c6740411275301f58de1c1876